testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/transaction/MaximumTimeoutTestCase.java

```diff
@@ -0,0 +1,213 @@
+package org.jboss.as.test.integration.transaction;
+
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.jboss.as.arquillian.api.ServerSetup;
+import org.jboss.as.arquillian.api.ServerSetupTask;
+import org.jboss.as.arquillian.container.ManagementClient;
+import org.jboss.as.controller.PathAddress;
+import org.jboss.as.controller.client.ModelControllerClient;
+import org.jboss.as.controller.client.helpers.ClientConstants;
+import org.jboss.as.controller.operations.common.Util;
+import org.jboss.as.test.integration.transactions.TestXAResource;
+import org.jboss.dmr.ModelNode;
+import org.jboss.logging.Logger;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import javax.annotation.Resource;
+import javax.transaction.HeuristicMixedException;
+import javax.transaction.HeuristicRollbackException;
+import javax.transaction.NotSupportedException;
+import javax.transaction.RollbackException;
+import javax.transaction.SystemException;
+import javax.transaction.TransactionManager;
+import javax.transaction.xa.XAException;
+import javax.transaction.xa.XAResource;
+import java.io.IOException;
+import java.util.List;
+
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.SUBSYSTEM;
+
+/**
+ * Tests for EAP7-981 / WFLY-10009
+ *
+ * @author istraka
+ */
+@RunWith(Arquillian.class)
+@ServerSetup({MaximumTimeoutTestCase.TimeoutSetup.class})
+public class MaximumTimeoutTestCase {
+
+    private static final String MESSAGE_REGEX = ".*\\bWARN\\b.*WFLYTX0039: The transaction timeout has been set to %d while the value is 0";
```

Comment:
I don't really like matching messages in a test as it's fragile. If it's necessary though we should only be matching the id. This would break with any internationalized languages.

Reply:
+1, we should just match on the ID if anything

Reply:
Having looked at this test more it seems meaningful to verify the two options in this integration test.

Reply:
The main issue is once translated interfaces are introduced this would break for anyone running the test with a translated local, e.g. `fr`. A pattern validating the number is okay, but we shouldn't use the English text.

Reply:
Is that even the case for tests? Do we run localized tests?

Reply:
The issue will show up in EAP where we do introduce localization. While CI should continue to work anyone building locally with a locale that's been translated will see this test fail. For example anyone in France with a French locale will see this test fail.